docs/source/usage.rst

```diff
@@ -351,6 +351,18 @@ You can use literal commas and spaces anywhere by escaping (:code:`\,` and :code
     >>> c.escape_whitespace
     'Hello World'
 
+Nested interpolations
```

Comment:
added the example for simple nested interpolation here as well.

Reply:
Also in a15623d